openapi.json

```diff
@@ -460,7 +460,7 @@
                                 "properties": {
                                   "amount": {
                                     "type": "number",
-                                    "example": 0.02
+                                    "example": 3
```

Comment:
Could you please pass logical example value here? If I see current & recommended, I wasn't able to relate this as example value.

Reply:
As mentioned by  @saltgen , he will create a separate pull-request to address this. Ticket [created](https://issues.redhat.com/browse/RHIROS-1224).
Merging this change.

Reply:
PR, https://github.com/RedHatInsights/ros-ocp-backend/pull/122

Reply:
@saltgen, #124 is the right PR, correct?

Reply:
@kgaikwad Yes, correct